### ISSUE TITLE
Support for NDJSON-LD?

### ISSUE THREAD

**Issue by wouterbeek** [rubensworks/jsonld-streaming-parser.js#64]
The Iconclass project was inventive enough to combine JSON-LD with NDJSON: http://iconclass.org/data/iconclass_20200529_skos_jsonld.ndjson.gz

Would it be doable to add support for NDJSON-LD?

**Comment by rubensworks** [comment]
This should definitely be doable. Would require a special `ndjson` mode that implicitly wraps a document within `[]`, and implicitly adds `,` after newlines.

**Comment by VladimirAlexiev** [comment]
Yeah, NDJSONLD is a thing and Ontotext added support for it in rdf4j: https://github.com/eclipse/rdf4j/issues/2840
